docs/nemoretriever-ocr.md

```
@@ -4,7 +4,7 @@
 -->
 # NeMo Retriever OCR Configuration Guide for NVIDIA RAG Blueprint (Early Access)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)